spec/en/reference.md

```diff
@@ -74,60 +76,87 @@ _Only files unique to GTFS-ride are defined below. Definitions for all other fil
+|   |  | * **0** - Entry contains complete ridership counts for the associated **stop_id** in the field(s) **boardings** and/or **alightings** as available. |
+|   |  | * **1** - Entry contains no ridership counts, but contains service cancellation data in **schedule_relationship**. |
 | schedule_relationship | Optional | The **schedule_relationship** field identifies whether service was scheduled and operated, or scheduled but not operated, or operated but not scheduled. If a trip is added it must have a **trip_id** that is not scheduled to run on that day and is unique among trips added on that day.  |
-|   |  | * **0** - (or empty) Service was scheduled and operated  |
+|   |  | * **0** - (or empty) Service was scheduled and operated. |
 |   |  | * **1** - whole scheduled trip was cancelled.  |
-|   |  | * **2** - whole scheduled trip was cancelled (but replaced with an added trip.)  |
+|   |  | * **2** - whole scheduled trip was cancelled (but replaced with an added trip).  |
 |   |  | * **3** - trip ran but this **stop_time** was cancelled.   |
-|   |  | * **4** - trip ran but this **stop_time** was cancelled (but replaced with a different stop.)  |
+|   |  | * **4** - trip ran but this **stop_time** was cancelled (but replaced with a different stop).  |
 |   |  | * **5** - whole trip was added. |
 |   |  | * **6** - whole trip was added (as a replacement.)  |
 |   |  | * **7** - this **stop_time** was added to a scheduled trip.    |
 |   |  | * **8** - this **stop_time** was added to a scheduled trip (replacing something cancelled).    |
+| boardings | Optional | The **boardings** field contains the number of boardings at (or nearest to, in the case of boardings inbetween stops ) the associated **stop_id** as collected by either automated or manual methods. Non-negative integer. |
+| alightings | Optional | The **alightings** field contains the number of alightings at (or nearest to, in the case of alightings inbetween stops) the associated **stop_id** as collected by either automated or manual methods. Non-negative integer. |
+| current_load | Optional | The **current_load** field contains the calculated percentage current load of a vehicle at the identified stop. The state at which **current_load** is measured, is specified by **load_type**; if no value is given in **load_type**, **current_load** is arriving load.  Non-negative integer.|
+| load_type | Optional | The **load_type** field specifies the state (arriving or departing) at which **current_load** is measured. If no value is given, **current_load** indicates arriving load |
+|   |  | * **0** - Arriving.  |
+|   |  | * **1** - Departing.  |
+| rack_down | Optional | The **rack_down** field indicated whether an external bike rack was deployed or remained down at the associated **stop_id**. This field should be used in conjunction with **bike_boardings** and **bike_alightings** if complete bike boarding and alighting counts are available. |
```

Comment:
might be better to be consistent in what tense is used. I would probably avoid past tense. for rack down where it says "field indicated whether..." it might be better to say indicates.